src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
 // write code here
+const listEmployee = document.body.querySelectorAll('li');
```

Comment:
It would be better to use `document.querySelectorAll('li')` instead of `document.body.querySelectorAll('li')` because `document.body` includes only the body of the HTML document, not the head. So if there are any `li` elements in the head (which is unusual, but possible), they would be ignored.